index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <a class="logo"> My Logo</a>
+    <img src="assets/Union.jpg" alt="menu-bar" />
+    <section class="intro-wrapper">
+      <h1 class="heading1">
+        I’m Linda Letaru<br />
+        Glad to see <br />you!
+      </h1>
+    </section>
+    <p class="content">
+      I’m a software developer! I can help you<br />
+      build a product , feature or website Look<br />
+      through some of my work and<br />experience! If you like what you see
+      and<br />
+      have a project you need coded, don’t<br />hestiate to contact me.
+    </p>
+    <div>
+      <br />
+      <p class="suggegiion-comment">LET’S CONNECT</p>
+      <br />
+      <div>
+        <h2 class="social media links">
+          <img src="assets/twittericon.svg" alt="bird-like-picture" />
+          <img src="assets/linkediniicon.svg" alt="recruiter website " />
+          <img src="assets/linkedinicon.svg" alt="recruiter website" />
+          <img src="assets/micon.svg" alt="letter-m-shaped word" />
+          <img src="assets/githubicon.svg" alt="cat-like-picture" />
+          <img src="assets/peaceicon.svg" alt="two-raised-figure" />
+        </h2>
```

Comment:
- [ ] good work implementing the social links however you need to group them in a ``` ul``` and wrap every logo in a ```li``` tag, every logo should be a link so also kindly consider wrapping every ```img``` in a ```a``` tag so it's a link